content/references/protocol-reference/transactions/transaction-types/checkcreate.md

```diff
@@ -42,7 +42,7 @@ Create a Check object in the ledger, which is a deferred payment that can be cas
 
 ## Error Cases
 
-- If the `Destination` is blocking incoming checks, the transaction fails with the result code `tecNO_PERMISSION`. _(Added by the [DisallowIncoming amendment][].)_
+- If the `Destination` account is blocking incoming Checks, the transaction fails with the result code `tecNO_PERMISSION`. _(Requires the [DisallowIncoming amendment][] :not_enabled:)_
```

Comment:
Three things.

1. I try to put names of ledger entries in capital letters to be clear it's a specific type of object. The term "checks" has a few meanings (invariant checks for example) so it's especially relevant with Checks.
2. See my comment above about rules for referencing amendments
3. The original sentence, as worded, uses a code-font word as a noun in the sentence, which breaks [Ripple style guide](https://ripplelabs.atlassian.net/wiki/spaces/DOCS/pages/2384134206/Writing+style+guide) rule 22:

> When referring to parts of a code object, always put the exact code in monospace/'code-style formatting' and use a word such as “field”, “value”, “object”, or “array” to identify it. Do not use the text of a field name as a stand-alone word in your sentence.
>
> _Our Style:_
> 
> > If the payment is successful, the transaction delivers the amount specified in the `Amount` field to the  `Destination` address.
> 
> Find a `DeletedNode` object in the transaction metadata.
> 
> _Not our style:_
> 
> > If the payment is successful, the transaction delivers the specified `Amount` to the `Destination`.
> 
> Find a `DeletedNode` in the `metaData`.
> 
> **Rationale:**
> Translation may struggle with sentences that use the word in code font because the code is the same for all users regardless of language.
>
> Also, a type word such as “field” or “value” helps to identify that this is an exact item from the code, in case people don’t notice the difference in typesetting.

Reply:
Could we create a public version of the style guide? (I don't think external contributors will be able to view the existing one)